tests/scripts/all.sh

```diff
@@ -644,6 +647,28 @@ pre_check_tools () {
     "$@" scripts/output_env.sh
 }
 
+# Set up an include directory for baremetal tests. Exclude headers that the
+# configuration says we shouldn't rely upon.
+pre_setup_baremetal_usr_include () {
```

Comment:
This setup should only be done in the few components that require it, not systematically.

Reply:
We no longer require setup as the files are added to git.